### ISSUE TITLE
Token error on upgrade to 49.9

### ISSUE THREAD

**Issue by kbrownlees** [metabase/metabase#42659]
### Describe the bug

I am attempting to upgrade our instance from v0.47.7 -> v0.49.9 and hitting a migration error.

We are not changing anything with our encryption key, but under 47.7 we are getting this error in the logs:

```
Cannot decrypt encrypted String. Have you changed or forgot to set MB_ENCRYPTION_SECRET_KEY?
```

Everything is working in 47.7 however as far as we can tell.

When attempting to upgrade to 49.9:

```
May 14, 2024 at 14:05 (UTC+12:00)	2024-05-14 12:05:15,281 ERROR metabase.core :: Metabase Initialization FAILED	metabase-test
May 14, 2024 at 14:05 (UTC+12:00)	liquibase.exception.CommandExecutionException: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset migrations/001_update_migrations.yaml::v49.2024-04-09T10:00:03::qnkhuat:	metabase-test
May 14, 2024 at 14:05 (UTC+12:00)	Reason: clojure.lang.ExceptionInfo: Unrecognized token '<omitted>': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')	metabase-test
May 14, 2024 at 14:05 (UTC+12:00)	at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 43] {:toucan2/context-trace [["resolve connection" {:toucan2.connection/connectable org.postgresql.jdbc.PgConnection}] ["resolve connection" {:toucan2.connection/connectable nil}]]}
```

I have also attempted a two step migration (47.7 -> 48 -> 49) and hit the same error.

### To Reproduce

1. Have an instance of 0.47.7
2. Attempt to upgrade to 0.49.9
3. Migration fails

### Logs

Migration error: 

https://gist.github.com/kbrownlees/4f390db9460e59cea7a81ed5e831b992

Start-up logs for 48.13:

https://gist.github.com/kbrownlees/b62de3366b1c06db8143281876eab816

### Information about your Metabase installation

```JSON
{
  "browser-info": {
    "language": "en-NZ",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:125.0) Gecko/20100101 Firefox/125.0",
    "vendor": ""
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.23+9",
    "java.vendor": "Eclipse Adoptium",
    "java.vendor.url": "https://adoptium.net/",
    "java.version": "11.0.23",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.23+9",
    "os.name": "Linux",
    "os.version": "5.10.215-203.850.amzn2.x86_64",
    "user.language": "en",
    "user.timezone": "Australia/Sydney"
  },
  "metabase-info": {
    "databases": [
      "postgres",
      "mysql"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "14.10"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.7.2"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2024-04-30",
      "tag": "v0.48.13",
      "hash": "7a5ccd8"
    },
    "settings": {
      "report-timezone": "Australia/Sydney"
    }
  }
}
```

### Severity

Blocking upgrade to 49

**Comment by paoliniluis** [comment]
we just fixed this today, expect it for 49.11

**Comment by vinceve** [comment]
Upgraded from 1.49.3 to 1.49.9 and got the same issue (although the error is slightly different)

```
tabase
2024-05-15 12:34:30,408 ERROR metabase.core :: Metabase Initialization FAILED
metabase
liquibase.exception.CommandExecutionException: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset migrations/001_update_migrations.yaml::v49.2024-04-09T10:00:03::qnkhuat:
metabase
Reason: clojure.lang.ExceptionInfo: Unexpected character ('Z' (code 90)): Expected space separating root-level values
metabase
at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 3] {:toucan2/context-trace [["resolve connection" {:toucan2.connection/connectable org.postgresql.jdbc.PgConnection}] ["resolve connection" {:toucan2.connection/connectable nil}]]}
metabase
    at liquibase.command.CommandScope.execute(CommandScope.java:253)
metabase
    at liquibase.Liquibase.lambda$update$0(Liquibase.java:245)
metabase
    at liquibase.Scope.lambda$child$0(Scope.java:186)
```

When will this issue be resolved and released? It's important for us as we are in the middle of a release and that this blocks our upgrade.

**Comment by paoliniluis** [comment]
We’re releasing the fix in 49.11
